src/compiler/crystal/tools/init.cr

```diff
@@ -31,13 +30,26 @@ module Crystal
           exit
         end
 
+        opts.on("--force", "force overwrite existing files") do
```

Comment:
It would be nice to have `-f` as as shorthand. Don't know about the other flag...